### PR TITLE
Count feature, related test and import optimisation

### PR DESCRIPTION
**Pull request by rkisdp**
Hi, @rednaks,

I have added the count feature and test for the same, and with that, I have also changed some other tests accordingly and removed unused imports from the file. Please have a look and give your feedback.

### CODE REVIEW

**Comment by rkisdp** [comment]
Hi, @rednaks, Can you please review the PR? and please let me know if you think I have made any mistake.

**Comment by rednaks** [comment]
Thank you, looks good for me, can you just update the README file for the count and none ?

Thanks !

**Comment by rkisdp** [comment]
Sure!

**Comment by rkisdp** [comment]
Hi, @rednaks, I have updated the README file and added some tests also. Please review this and let me know if you want me to add something else.

**Comment by rkisdp** [comment]
Hi @rednaks please review this

**Comment by rkisdp** [comment]
Hi, @rednaks, Can you please review the changes and let me know if you want any more here? I want to contribute more.

File: tests/test_django_async_orm.py [hunk 1]
```diff
@@ -96,29 +81,29 @@ async def test_async_in_bulk(self):
...
         created = await TestModel.objects.async_create(name="to delete")
         all_created = await TestModel.objects.async_all()
-        self.assertEqual(len(all_created), 3)
+        count = await all_created.async_count()
+        self.assertEqual(count, 3)
 
         await all_created.async_delete()
         all_after_delete = await TestModel.objects.async_all()
-        self.assertEqual(len(all_after_delete), 0)
+        count = await all_created.async_count()
```
Comment (rednaks) [inline tests/test_django_async_orm.py:91]: you should use `all_after_delete` rather than `all_created`

File: tests/test_django_async_orm.py [hunk 2]
```diff
@@ -198,9 +185,22 @@ async def test_async_prefetch_related(self):
...
+            qs = await qs.async_first()
+            self.assertEqual(qs.name, "setup 1")
+
+        async def test_async_order_by_descending(self):
+            qs = await TestModel.objects.async_all()
+            qs = await qs.async_order_by('-name')
+            qs = await qs.async_first()
+            self.assertEqual(qs.name, "setup 2")
+
+        await test_async_order_by_ascending(self)
```
Comment (rednaks) [inline tests/test_django_async_orm.py:202]: would be better to split then into two different tests, always one assert per unit test

File: tests/test_django_async_orm.py [hunk 3]
```diff
@@ -41,26 +33,19 @@ async def test_async_get(self):
         self.assertEqual(result.name, "setup 1")
 
     @tag('ci')
-    async def test_async_create(self):
-        result = await TestModel.objects.async_create(name="test")
-        self.assertEqual(result.name, 'test')
-
-    @tag('ci')
-    async def test_async_bulk_create(self):
```
Comment (rednaks) [inline tests/test_django_async_orm.py:49]: why did you remove this test ?! we need to test `async_bulk_create` !!

File: README.md [hunk 4]
```diff
@@ -130,8 +132,6 @@ You can still call `Model.object.async_raw()` but you will be unable to access t
 | methods                    | supported  | comments |
 |----------------------------|------------|----------|
 | `Model.async_save`                      | ❌ |  |
-| `Model.async_update`                    | ❌ |  |
-| `Model.async_delete`                    | ❌ |  |
```
Comment (rednaks) [inline README.md:134]: why did you remove this ?

File: tests/test_django_async_orm.py [hunk 5]
```diff
@@ -41,26 +33,19 @@ async def test_async_get(self):
...
-        ])
+    async def test_async_get_or_create(self):
 
-        self.assertEqual(len(objs), 2)
+        async def test_async_get_or_create_on_obj_get(self):
+            obj = await TestModel.objects.async_get_or_create(name="setup 1")
+            self.assertEqual(obj[1], False)
 
-    @tag('dev')
-    async def test_async_bulk_update(self):
```
Comment (rednaks) [inline tests/test_django_async_orm.py:58]: ?!

**Comment by rednaks** [comment]
Thank you for your PR, but I don't know why you removed so many tests ... please revert them and make the changes requested in the comments.

Thank you

**Comment by rkisdp** [comment]
Hi, @rednaks did you check this? any feedback?
